dsl/src/runtime/GameEnvironment.java

```diff
@@ -100,6 +97,20 @@ public void loadTypes(IType[] types) {
         }
     }
 
+    @Override
+    public void loadFunctions(ScopedSymbol[] functions) {
```

Comment:
`ScopedSymbol[]`: Willst Du in Java wirklich mit Arrays arbeiten und nicht mit Datentypen, die mehr Komfort anbieten?

Reply:
Angepasst.